src/styles/blocks/watch.scss

```
@@ -0,0 +1,76 @@
+.watch {
+  position: relative;
+
+  @include circle($watch-size);
+  @include center-block;
+
+  background-color: $bgc;
+  border-radius: 50%;
+  border: 15px double $yellowColor;
+
+  &__center {
+    @include circle($center-size);
+
+    background-color: #001064;
+    border-radius: 50%;
+
+    z-index: 1;
+  }
+
+  &__hour-hand {
+    position: absolute;
+    bottom: 50%;
+
+    width: 10px;
+    height: 120px;
+    background-color: $yellowColor;
+
+    @include animation-styles;
+    animation-duration: 43200s;
```

Mb will be better if we make one hour arrow & one minute arrow?
```suggestion
    animation-duration: 3600s;
```